ios/Rakefile

```diff
@@ -1,5 +1,5 @@
 FULL_TESTS = ENV['FULL_TESTS'] != 'false'
-BUILD_SDK = ENV['BUILD_SDK'] || 'iphonesimulator13.7'
+BUILD_SDK = ENV['BUILD_SDK'] || 'iphonesimulator14.4'
```

Comment:
Same as before – is it worth us making this 14.5?

Reply:
This was to align with what we are using for the Backpack iOS app so not sure if it would cause issues if there are mis aligned

Reply:
When we upgrade to xcode 12.5 then lets also change this to iOS 12.5